test/examples/17-resizable-handles.jsx

```diff
@@ -4,6 +4,7 @@ import RGL, { WidthProvider } from "react-grid-layout";
 
 const ReactGridLayout = WidthProvider(RGL);
 
+window.asdf = RGL;
```

Comment:
?

Reply:
🙃 sorry, just testing stuff from console on the window. Shouldn't have been committed.